### DISCUSSION TITLE
onSelect break user selection from list

### DISCUSSION THREAD

**Discussion by pasadyaguy** [afconsult/quill-mention#326] [Q&A]
I saw a few other onSelect issues but this one is different from what I was reading in the others. 

When I do not use onSelect, I can search for a user and select the user and everything is normal. 

When I add onSelect, I can still search for a user, however when the user is selected, it doesn't show on the editor that it was selected. It just stops where I finished typing. It DOES however output the correct info in the onSelect method.

Here is a example [https://stackblitz.com/edit/angular-quill-mention-bug?file=src%2Fapp%2Fapp.component.ts](https://stackblitz.com/edit/angular-quill-mention-bug?file=src%2Fapp%2Fapp.component.ts)

Notice how when I select the item when using onSelect, it doesn't actually select them in the editor, but it logs to the console.
![quill-mention-on-select-issue](https://user-images.githubusercontent.com/19986009/86289160-31f97f80-bbb9-11ea-9f03-24e72cd255b7.gif)

**Comment by jamesthomsondev** [comment]
@smithbrianscott Were you able to resolve this? I have a similar issue that's just started happening and I have no idea where it stems from. The select seems to work, but the content inserted is empty.
